README.md

```diff
@@ -1141,6 +1142,11 @@ In either case:
       ]
 
       # good
```

Comment:
I was looking at one of our other class blocks below and it also does the re-declaration thing, so that's why I did it this way. I can make it a different block if we feel that'll be cleaner.